group_vars/development

```diff
@@ -1,12 +1,13 @@
 mysql_root_password: devpw
 
+deploy_user: vagrant
+
 wordpress_sites:
-  - site_name: example.dev
+  example.dev:
     site_hosts:
       - example.dev
+    repo: https://github.com/roots/bedrock.git
```

Comment:
I think this `repo` variable is only referenced in the `deploy.yml` playbook, and `deploy.yml` isn't used for development. Development continues using Vagrant sync folders for its bedrock.

This means there is a little less parity between development and production environments, in how they are provisioned. But I'm guessing it's better than problems that might arise by trying for parity on this issue.